packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -409,6 +425,11 @@ export default class BuySellStore extends BaseStore {
         this.should_show_popup = should_show_popup;
     }
 
+    @action.bound
+    setShouldShowRateChangedPopup(should_show_rate_changed_popup) {
```

Comment:
```suggestion
    setShowRateChangedPopup(show_rate_changed_popup) {
```
Nit: a shorter name is preferable. Please update the other place we reference this as well

Reply:
Done, thx for notice!

Reply:
> Nit: a shorter name is preferable. Please update the other place we reference this as well

Done, thx for notice!